### PR TITLE
[9.0] Add stock_scanner_shipping module

### PR DESCRIPTION
**Pull request by Garamotte**
This PR is based on https://github.com/OCA/stock-logistics-barcode/pull/40

It adds a scenario module : Shipping/Delivery

### CODE REVIEW

**Comment by jgrandguillaume** [comment]
Hi,

Great work ! Good to split one module per scenario. Just a little remark:

I saw you have a common readme for all scenarios modules. It would be nice to have instead a explanation of the scenario it-self.

Regards,

Joël

**Comment by jgrandguillaume** [comment]
Hi,

Great work ! Good to split one module per scenario. Just a little remark:

I saw you have a common readme for all scenarios modules. It would be nice to have instead a explanation of the scenario it-self.

Regards,

Joël

**Comment by jgrandguillaume** [comment]
:+1:  After discussion :)

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Functional review

**Comment by pankk** [comment]
:+1:

File: stock_scanner_shipping/__init__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```
Comment (lasley) [inline stock_scanner_shipping/__init__.py]: Please use new header

File: stock_scanner_shipping/__init__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+
+
+# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Comment (lasley) [inline stock_scanner_shipping/__init__.py]: Remove

File: stock_scanner_shipping/__init__.py (hunk 1 above)
Comment (Garamotte) [reply stock_scanner_shipping/__init__.py]: **init**.py file emptied.

**Review by lasley — APPROVED** [review]

**Pull request merged**